code/modules/surgery/organs/ears.dm

```diff
@@ -82,6 +82,11 @@
 		ear_owner.dna.species.mutant_bodyparts -= "ears"
 		ear_owner.update_body()
 
+/obj/item/organ/ears/cat/modder
+	name = "gene modder's ears"
```

Comment:
These are fine to leave alone.

Reply:
i wanna leave it like that because im intending to add more tails and ears and it would make zero sense if they were all labeled as cat ears whenever you chopped them off

Reply:
You should be adding a different organ for each of the modder's ear types.

Reply:
ill just axe them for now, when the sprite accessory split comes in ill get extra ears and tails in